api/filter/rate_limit.proto

```diff
@@ -2,6 +2,19 @@ syntax = "proto3";
 
 package envoy.api.v2.filter;
```

Comment:
Should we put network filters in a network subdirectory and http filters in an http subdirectory? Seems like it would be cleaner? If you think that is good do you want to just move stuff around and then do the cleanup required in envoy for HTTP connection manager?

Reply:
PTAL.. shuffled stuff around

Reply:
I think changing package names can be considered an API breaking change. I don't think we can do this with HTTP connection manager..

Reply:
if its just envoymanager using it, and may be @timperret, we could do the right thing now than later?

Reply:
@htuch is it really breaking? Since we aren't using Any yet, and just Struct, it seems like this would be a compile only issue and not a wire issue? If so we should clean it up?

Reply:
It's breaking in the sense that a `.proto` for the filter config that wasn't encoded via `Struct` would break I think, but OK, I don't mind relaxing this one.